packages/types/src/index.ts

```diff
@@ -115,23 +124,23 @@ export interface MultiUsersSelect extends Action {
 export interface StaticSelect extends Action {
   type: 'static_select';
   placeholder?: PlainTextElement;
-  initial_option?: Option;
-  options?: Option[];
+  initial_option?: PlainTextOption;
```

Comment:
I've verified that all the type changes for these block types (static_select/external_select, multi_static_select/external_select, overflow) are correct.